edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -56,9 +53,8 @@
 
       if (!attempt_proctored) {
         var msg = gettext(
-          "Are you sure you want to take this exam without proctoring? " +
-          "This will make you no longer eligible to earn credit for this course."
-        )
+          "You will no longer be eligible to use this course for academic credit."
```

Comment:
Sorry -- my comment on PR#17 wasn't clear enough. The original first sentence should remain as well: 
"Are you sure you want to take this exam without proctoring?
You will no longer be able to use this course for academic credit."